spec/models/user_spec.rb

```diff
@@ -0,0 +1,182 @@
+require 'spec_helper'
+
+describe User do
+
+    describe "needs_shave_or_trimming" do
+        it "should be false when status is present & shaved_at date is less than 3 days old" do
+            @user = User.create(first_name: "test", last_name: "user", email: Random.email, :gender => "male", :status => ["shaved", "trimmed", "massaged_and_trimmed", "unshaved"].sample, :shaved_at => DateTime.now)
```

Comment:
Move setup steps to `let` blocks. Try to keep `it` blocks as clean as possible, so that it only contain assertions.